packages/uswds-core/src/styles/_deprecated.scss

```diff
@@ -18,4 +18,4 @@
 */
 
 // Deprecated in 3.0.0
-$output-all-utilities: true;
\ No newline at end of file
+$output-all-utilities: true !default;
```

Comment:
Added a default flag here, since even though this is no longer used, we don't want to break compile if it's included in settings